modules/authentication/src/handlers/local.ts

```diff
@@ -734,13 +738,18 @@ export class LocalHandlers implements IAuthenticationStrategy {
       user,
     );
     const tokenPromise = Token.getInstance().deleteOne(verificationTokenDoc);
+    await this.grpcSdk.state!.setKey(
```

Comment:
I don't like how these tokens are just plainly dumped straight into Redis, but I guess there's no other way around it as Redis does not support expiring individual hash keys.

Can we at the very least prefix these with an additional `_verifiedToken` or something?
That would result in Redis storing these as `authentication_verifiedToken_XYZXYZXYZ`.